account-admin/byob.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "Bring Your Own S3 Bucket"
+description: "Configure your own AWS S3 bucket to work with Rive"
+---
+
+## What is a Custom S3 Bucket?
+
+A custom S3 bucket allows Enterprise customers to store their Rive assets in their own AWS environment instead of Rive's default storage. This premium feature gives you greater control over your data, allowing you to:
```

Comment:
```suggestion
A custom S3 bucket allows Enterprise customers to store their Rive file and asset data in their own AWS environment instead of Rive's default storage. This premium feature gives you greater control over your data, allowing you to:
```

Reply:
Yeah I was torn by this too, but then I thought that a File is an asset in some way..? But, since you noticed this too, I'll change it up